openedx/features/edly/tests/test_models.py

```diff
@@ -75,3 +81,37 @@ def test_edly_sub_organization_post_update_receiver(self):
 
         assert len(edly_sub_organization_data) == 1
         assert edly_sub_organization_data[0].edly_organization.name == edly_organization.name
+
+
+class EdlyUserProfileTests(TestCase):
+    """
+    Tests for EdlyUserProfile Model.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(EdlyUserProfileTests, cls).setUpClass()
+        cls.user = UserFactory(password="test")
+
+    def test_edly_user_profile_post_save_receiver(self):
+        """
+        Test EdlyUserProfile model object creation.
+        """
+
+        edly_user_profile = EdlyUserProfileFactory(user=self.user)
+        edly_user_profile_data = EdlyUserProfile.objects.filter(user=edly_user_profile.user)
+        assert len(edly_user_profile_data) == 1
+
+    def test_multiple_sub_organizations(self):
+        """
+        Test EdlyUserProfile model multisite sub_organizations.
+        """
+
+        edly_sub_organizations = [EdlySubOrganizationFactory() for __ in range(3)]
+        edly_user_profile = EdlyUserProfileFactory(user=self.user)
+        edly_user_profile.edly_sub_organizations.add(*edly_sub_organizations)
+
+        edly_user_profile_data = EdlyUserProfile.objects.filter(user=edly_user_profile.user)
+
+        assert len(edly_user_profile_data) == 1
```

Comment:
there is a better way to do this. If you just need to check that there is some data in the returned queryset, use `edly_user_profile_data.exists()`. If you need to verify it's just one entry, use `edly_user_profile_data.count()`. This goes for all such assertions.

Reply:
count/exists can also do the same thing but as in all previous unit tests, we went with `len` based check so keep things consistent we are using this here.

Reply:
I think using db based count is cleaner way.
```python
assert edly_user_profile_data.count() == 1
```

Reply:
Done